src/server/components/dashboard/listsItems/controllers.ts

```diff
@@ -337,13 +342,14 @@ export async function handlePinListItem(
     }
 
     return listItem;
-  } catch (e: any) {
-    logger.error(`deleteListItem Error ${e.message}`);
+  } catch (error) {
+    logger.error(`deleteListItem Error ${(error as Error).message}`);
 
     throw new Error(`Failed to ${isPinned ? "pin" : "unpinned"} item`);
```

Comment:
Should just be able to throw a new error (and chain the original error) and rely on the error handler middleware to log it out.